src/App.js

```diff
@@ -14,11 +14,70 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    values: [],
+  }
+
+  render() {
+    const { values } = this.state;
+
+    return (
+      <div className="App">
+        <h1>
+          { values.length > 0
+            ? (
```

Comment:
```suggestion
          { value ? (
```
If you will have `value` instead of `values` you may use it like this